scripts/test-local.sh

```diff
@@ -1,13 +1,16 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
This way of checking for unset env vars is a bashism, are we OK to require bash here?

Reply:
Humph, looks like the docker containers may not have `/bin/bash`...